### ISSUE TITLE
Can this suit stealth navigationStart

### ISSUE THREAD

**Issue by tzbo** [apify/fingerprint-suite#95]
I find some bot detect window.performance.timing.navigationStart which is used to find which browser contextes are created at same time.

Please help to stealth this property.

**Comment by tzbo** [comment]
[Here](https://www.zenrows.com/blog/bypass-cloudflare#how-to-bypass-cloudflare)

**Comment by barjin** [comment]
Thank you for filing the issue @tzbo!

I will join @wireguard-dev in asking for more details here. Could you please share your setup / show us what issue are you dealing with specifically? AFAIAC, Playwright does not behave any different from regular Chromium/Firefox instance in this way.

**Comment by tzbo** [comment]
Sorry, I don't know how does cloudflare detect this feature. Maybe the start time of browser. Because we create many contextes at the same time.

**Comment by barjin** [comment]
Sorry for leaving this issue open with no resolution for so long. After further inspection, we didn't find any correlation between the timing properties and getting blocked by antibot services (while using fingerprint-suite). 

This project tries to hide headless-only features in browsers (and randomize parts of the browsers' fingerprints so that the antibot services cannot identify individual instances of your scrapers. The timing information seems to contain no specific information which could help them do so. 

If you are suspicious you might be getting blocked based on this properties' value, try slowing down your script a bit - so that it seems more human-like to the website.

As there is nothing much we can do with this in this library, I'll close this issue now as `wontfix` - but as always; feel free to reopen in case of any questions / ideas. Thanks :)
